exasol/toolbox/util/release/changelog.py

```diff
@@ -112,7 +132,7 @@ def update_changelogs_for_release(self) -> None:
         """
         Rotates the changelogs as is needed for a release.
 
-          1. Moves the contents of the `unreleased.md` to the `changes_<version>.md`
+          1. Moves the changes from the `unreleased.md` to the `changes_<version>.md`
```

Comment:
In this case, I would prefer "contents" to be honest.
Alternatively, we could say "text" or "description(s)" or "change descriptions".

But actually, I think, "content" is most generic and still most correct.